setup.py

```diff
@@ -45,7 +45,7 @@ def find_scripts(script_dir, prefix):
         "astropy~=4.0,>=4.0.2",
         'ctapipe>=0.10.1',
```

Comment:
Why >=0.10.1 here and 0.10.2 in ci.yml?

Reply:
0.10.2 is not yet on conda-forge

Reply:
Now it is.

Reply:
can we update this one with the latest ctapipe version (v0.10.3) to avoid problems?

Reply:
can we update this one with the latest ctapipe version (v0.10.3) to avoid problems?

Reply:
Yes, sure.